### ISSUE TITLE
Scipy 0.18 produces a ValueError

### ISSUE THREAD

**Issue by samuelstjean** [ecaruyer/phantomas#38]
It seems like they changed something in one of their function definition, as I get 

~~~bash
Reading geometry description...
Traceback (most recent call last):
  File "/home/samuel/.local/bin/phantomas_dwis", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/home/samuel/git/phantomas/scripts/phantomas_dwis", line 98, in <module>
    f = FiberSource(control_points, tangents=tangents_mode)
  File "/home/samuel/git/phantomas/phantomas/geometry/fiber.py", line 42, in __init__
    self._create_from_control_points(control_points, tangents, scale)
  File "/home/samuel/git/phantomas/phantomas/geometry/fiber.py", line 94, in _create_from_control_points
    scale * np.vstack((control_points[:, 0], derivatives[:, 0])).T)
  File "/home/samuel/.local/lib/python2.7/site-packages/scipy/interpolate/interpolate.py", line 643, in __init__
    raise ValueError("%s must be between 0 and %s" % (axis, c.ndim-1))
ValueError: 0 must be between 0 and 0
~~~

Same thing happen in 0.18.1 for various bundled valid txt control files with phantomas. 
Downgrading to 0.17 works perfectly well and does not raise any error message though, so it can be worked around easily.

**Comment by ecaruyer** [comment]
You are right, indeed I am having the same issue here on my box. Seems like newer scipy versions are offering more choice for the piecewise polynomial fit method we want to use. I will dig into this, probably no sooner than 11/21st anyway.

**Comment by ecaruyer** [comment]
Porblem is described here: https://docs.scipy.org/doc/scipy/reference/release.0.18.0.html#scipy-interpolate

As suggested, I will replace with scipy.interpolate.BPoly.from_derivatives

**Comment by ecaruyer** [comment]
Fixed by #39
